### PR TITLE
Implementation status added

### PR DESCRIPTION
**Pull request by naitoh**
* Implementation status added
* Name and Description field updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
-## Name
-
-Red Chainer
+A flexible framework for neural network for Ruby
 
 ## Description
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/chainer`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+It ported python's Chainer with Ruby.
```
Comment (hatappi) [inline README.md]: When the user wants to know something about chainer, how about linking so that it can be found easily.

Comment (naitoh) [reply README.md]: Thank you for your comment.
I added the link of Chainer.

**Comment by naitoh** [comment]
I fixed it.
Thank you.

**Comment by hatappi** [comment]
Than you for fixed!

**Pull request merged**